pub mod part1;

use std::error::Error;

pub fn main() -> Result<(), Box<dyn Error>> {
    part1::main()
}
